huaweicloud/services/servicestage/resource_huaweicloud_servicestage_component_instance.go

```diff
@@ -1446,16 +1446,20 @@ func resourceComponentInstanceDelete(ctx context.Context, d *schema.ResourceData
 	return nil
 }
 
-func componentInstanceRefreshFunc(c *golangsdk.ServiceClient, instance string) resource.StateRefreshFunc {
+func componentInstanceRefreshFunc(c *golangsdk.ServiceClient, jobId string) resource.StateRefreshFunc {
 	return func() (interface{}, string, error) {
-		resp, err := jobs.Get(c, instance)
+		opt := jobs.ListOpts{
+			Limit: 50,
```

Comment:
why Limit 50 is enough? do we need to do loop request to query all itmes

Reply:
@niuzhenguo List method will get all tasks with pagination in SDK